docs/section18/Rule_18-21.md

```diff
@@ -0,0 +1,52 @@
+# HVAC_SystemZoneAssignment - Rule 18-21
+**Schema Version:** 0.0.22  
+
+**Rule ID:** 18-21
+**Rule Description:** For baseline HVAC systems 1, 2, 3, 4, 9, 10, 11, 12, and 13, each HVAC zone or thermal block shall be modeled with its own HVAC system, , with the exception of baseline system 5 or 7 serving laboratory spaces.
+
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** G3.1.1.a.4
+
+**Applicability:** 
+
+1. System type is 1,2,3,4,9,10,11,12, or 13
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each system serves an entire floor
+
+**Data Lookup:**   
+**Function Call:** 
+
+1. get_baseline_system_types()
+2. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()
+3. baseline_system_type_string_compare()
+
+
+## Rule Logic:  
+- get the list of all the baseline system types that exist in the baseline model: `baseline_system_types = get_baseline_system_types(B-RMD)`
+- get the list of HVAC systems and the zones that they serve: `hvac_systems_zones_terminal_units = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B-RMI)`
+
+
+- loop through the systems types in baseline_system_types: `for system_type in baseline_system_types:`
+  - create a variable result that is an enum and set it to NOT_APPLICABLE: `result = NOT_APPLICABLE`
+	- check whether the system type is one of system type 1,2,3,4,9,10,11,12, or 13: `if any(baseline_system_type_string_compare(system_type, sys_type, false) == TRUE for sys_type in ["SYS-1","SYS-2","SYS-3","SYS-4","SYS-9","SYS-10","SYS-11","SYS-12","SYS-13"]):`
+		- loop through the hvac systems of this type and do the rule check: `for system_id in baseline_system_types[system_type]:`
+			- now we need a list of the zones served by the hvac system: `zones_served_by_system = hvac_systems_zones_terminal_units[system_id]["ZONE_LIST"]`
+			- there should only be one zone attached to the system for the system to pass: `if len(zones_served_by_system) == 1:`
+			  - this system passes, set result to PASS: `result = PASS`
+			- else the system fails: `else:`
+			  - set result to fail: `result = FAIL`
+
+**Rule Assertion - Zone:**
+
+  - Case 1: The system is not one of the tested system types, the rule is not applicable: `NOT_APPLICABLE if result == NOT_APPLICABLE`
```

Comment:
I don't think we need to produce Not Applicable assertaions

Reply:
changed rule logic to have a clear applicability check instead of NOT_APPLICABLE result